lunarity/Cargo.toml

```diff
@@ -6,7 +6,9 @@ license = "GPL-3.0"
 repository = "https://github.com/maciejhirsz/lunarity"
 
 [dependencies]
-toolshed = { version = "0.4" }
+lunarity-ast = { path = "../ast" }
+lunarity-lexer = { path = "../lexer" }
+lunarity-parser = { path = "../parser" }
+toolshed = "0.4"
```

Comment:
The only reason this is here is because `benches/second-price-auction.rs` requires it.

the alternative would be to `pub extern crate from lexer`

putting it into `dev.dependencies` doesn't work.

Reply:
Make it `dev-dependencies`, then it works :)